docs/content.zh/release-notes/flink-1.17.md

```diff
@@ -0,0 +1,203 @@
+---
+title: "Release Notes - Flink 1.17"
+---
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+# Release notes - Flink 1.17
+
+These release notes discuss important aspects, such as configuration, behavior, or dependencies,
```

Comment:
```suggestion
These release notes discuss important aspects, such as configuration, behavior or dependencies,
```
not sure, however it looks like no need for comma here...
I try to follow this rule https://www.grammarly.com/blog/comma-before-or/